docs/UserGuide.md

```diff
@@ -77,156 +105,254 @@ Format: `help`
+New applicant added: Bob; 
+Phone: 99999999; 
+Email: bob@example.com; 
+Job: Data Analyst;
+Round: Interview; 
+Skills: [Java][Pandas][Python]
 ```
 
 [Back to top <img src="images/back-to-top-icon.png" width="25px" />](#table-of-contents)
 
 ---
 ### Listing all applicants : `list`
 
-Show a list of all job applicants, along with an overview of each applicant.
+Allows you to list out all job applicants, along with an overview of each applicant.
+
+<br>
 
 Format:
 ```
 list
 ```
+
+<br>
+
 Example:
 ```
 list
 ```
 Sample Output:
 ```
 List of applicants:
-1. Steve Jobs
-Job Applied: Product Designer
-Skills: Product Design
-Phone Number: 99999999
 
-2. Elon Musk
-Job Applied: Software Developer
-Skills: Pandas, Python
-Phone Number: 88888888
+Name: Steve Jobs;
+Phone: 99999999;
+Email: stevejobs@apple.mail;
+Job: iOS Developer;
+Round: Hired;
+Skills: Swift
+
+Name: Elon Musk;
+Phone: 88888888;
+Email: elonmusk@tesla.com;
+Job: Software Engineer;
+Round: Technical Interview;
+Skills: Java JavaFX Gradle;
 ```
 
 [Back to top <img src="images/back-to-top-icon.png" width="25px" />](#table-of-contents)
 
 ---
 ### Viewing a specific applicant : `view`
 
-Allows the user to view an overview of a specific applicant, specified by applicant's full name (exact match)
+Allows you to view an overview of a specific applicant, specified by an applicant's full name (exact match)
+
+<br>
 
 Format:
 ```
-view n/NAME
+view NAME
 ```
 * The search is case-insensitive. e.g `hans` will match `Hans`
 * Only the name is searched.
 * Only full name will be matched e.g. `Han Lee` will not match `Han`
 * Only exact full name with correct spacing will be matched e.g. `HanLee` will not match `Han Lee`
 
+<br>
+
 Example:
 ```
-view n/Steve Jobs
+view Steve Jobs
 ```
 Sample Output:
 ```
-Here’s an overview of Steve Jobs:
-Phone Number: 99999999
-Job Applied: CEO of Apple
-Skills: Product Design  
+Name: Steve Jobs;
+Phone: 99999999;
+Email: stevejobs@apple.mail;
+Job: iOS Developer;
+Round: Hired;
+Skills: Swift
 ```
 
 [Back to top <img src="images/back-to-top-icon.png" width="25px" />](#table-of-contents)
 
 ---
 ### Searching a specific applicant : `search`
 
-Allows the user to view an overview of a specific applicant, specified by applicant's first or last name 
+Allows you to view an overview of a specific applicant, specified by **either** an applicant's name, job, round or skills.
+
+<br>
 
 Format:
 ```
-search n/NAME
+search [n/NAME]... [j/JOB]... [r/ROUND]... [s/SKILL]...
 ```
 * The search is case-insensitive. e.g `hans` will match `Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
 * First name/Last name will be matched e.g. `Han` will match `Han Lee`
+* Only full words will be matched e.g. `Han` will not match `Hans`
+* Search will not work with multiple prefixes. e.g `search n/Alex j/Interview` will only yield results of `search n/Alex`
```

Comment:
which version of search are we suppose to include?

Reply:
Yup I have updated UG to work with your latest PR.